pokemongo_bot/inventory.py

```diff
@@ -1134,9 +1139,16 @@ def update_web_inventory(self):
         if not os.path.exists(web_inventory):
             self.init_inventory_outfile()
             
-        with open(web_inventory, "r") as infile:
-            json_inventory = json.load(infile)
-            infile.close()
+        try:
```

Comment:
Yeah, meant the whole lot - Just a thought! Only really seeing the difference as some read, some write. Either way, not a bit deal - was just seeking to reduce redundancy.